Source/Core/Core/CoreTiming.cpp

```diff
@@ -479,8 +479,8 @@ void Idle()
 
 	idledCycles += DowncountToCycles(PowerPC::ppcState.downcount);
 	PowerPC::ppcState.downcount = 0;
-
-	Advance();
+	if(_M_ARM_32) //Temp Fix
```

Comment:
@lioncash : Reverse the situation, it would never happen (at least not for as long) on the JIT64